src/openapi/transport/core.js

```diff
@@ -75,6 +80,8 @@ function Transport(baseUrl, options) {
     this.baseUrl = baseUrl;
     this.language = options && options.language;
     this.defaultCache = options && typeof options.defaultCache === 'boolean' ? options.defaultCache : DEFAULT_CACHE;
+

```

Comment:
I think this should be outside the class so if you create 2 transports they share the same counter..